#VRML_SIM R2019b utf8
WorldInfo {
  info [
    "The user drives a Supervisor by the Keyboard which drives slaves robots by using an emitter device."
  ]
  title "Omniscience"
}
Viewpoint {
  orientation 0.002913609200257083 0.9378267503227659 0.3470914825524659 3.0714491299316133
  position 0.4166985373420223 3.5797148993968313 -3.9685738209039005
}
TexturedBackground {
}
TexturedBackgroundLight {
}
RectangleArena {
  translation -0.07 0 0
}
DEF ThymioII_1 Thymio2 {
  hidden position_0_0 -2.8950337872182505e-10
  hidden position_0_1 -2.88714965374781e-10
  translation -0.44916777003186314 -6.759124577921521e-05 0.12497607155438034
  rotation 0.004337939679489206 -0.9999807628846435 -0.004433524555397927 -1.5926273825708932
  name "ThymioII_1"
  controller "slave"
}
DEF ThymioII_2 Thymio2 {
  hidden position_0_0 -1.047334826553004e-12
  hidden position_0_1 -6.714314875759559e-13
  translation -0.4418670000595184 -6.716819202609581e-05 -0.12552199999194286
  rotation 0.004349502267756946 -0.9999807124758782 -0.004433565862594347 -1.5899754155158101
  name "ThymioII_2"
  controller "slave"
}
DEF ThymioII_3 Thymio2 {
  hidden position_0_0 -9.987277805389646e-13
  hidden position_0_1 -6.234646532479364e-13
  translation -0.4483830000665733 -6.752721568172027e-05 -0.36351399999180806
  rotation 0.0043495094188286745 -0.9999807124124553 -0.004433573151985887 -1.5899754155791357
  name "ThymioII_3"
  controller "slave"
}
Robot {
  children [
    Emitter {
    }
  ]
  name "emitter"
  controller "driver"
  supervisor TRUE
}
